sitemedia/scss/components/_results.scss

```diff
@@ -103,6 +96,8 @@ section#document-list {
 
         // styles specific to transcription
         .transcription {
+            @include typography.transcription;
```

Comment:
I think we should apply the font style based on the script of the associated language, but I'm not indexing that in Solr yet and there are plenty of documents with transcription that don't have language metadata yet.

Is it a problem if we apply this to transcription with arabic text? There are only a handful, but there are some. Test cases: PGPID 4473, 19011, 1242, 8273, 8288

Maybe we can assume hebrew if script is hebrew or unset (once we add that), so that known-arabic documents won't get the wrong font.